skumm00-proj.md

```diff
@@ -0,0 +1,37 @@
+---
+name: "@Skumm00"
+project: "Robot Arm"
+---
+
+#  Project Name - ArduinArm
+
+##  Summary
+
+My project is going to be a custom-controlled, Arduino-powered, robot arm.
```

Comment:
Could you please elaborate a bit more here?

Some prompts you may wish to answer in this section:
What excites you about building this project? Have you built similar hardware projects before? Why do you want to build this project?